core/graph/src/test/java/uk/gov/gchq/gaffer/graph/hook/UpdateViewHookTest.java

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
I think I would be tempted to put all of these UpdateViewHook test classes in the one UpdateViewHookTest class.